docs/develop/go/temporal-nexus.mdx

```diff
@@ -192,6 +192,20 @@ var HelloOperation = temporalnexus.NewWorkflowRunOperation(service.HelloOperatio
 Workflow IDs should typically be business meaningful IDs and are used to dedupe Workflow starts.
 For the `HelloOperation`, `input.ID` is passed as part of the Nexus Service contract.
 
+:::tip
+
+[Pre-release feature](/evaluate/development-production-features/release-stages#pre-release): Attaching multiple Nexus callers to an underlying handler Workflow.
+
+When using `temporalnexus.NewWorkflowRunOperation` to start a handler Workflow with a [Workflow-Id-Conflict-Policy](/workflows#workflow-id-conflict-policy) of Use-Existing
+all caller Workflows will have completion Callbacks attached to the underlying handler Workflow.
+The initial caller Workflow will be linked from the handler's [WorkflowExecutionStarted](/references/events#workflowexecutionstarted) event and
+additional callers will be attached to the handler's [WorkflowExecutionOptionsUpdated](/references/events#workflowexecutionoptionsupdated) event.
+
+When a handler Workflow uses [Continue-As-New](/workflows#continue-as-new) the existing Nexus completion Callbacks and bi-directional Links will be copied to the new Workflow-Execution
+and the previous Workflow-Execution's completion Callback will be left in the Standby state indefinitely.
```

Comment:
This is not accurate.
- For continue as new, all callbacks are copied over to new workflow, but only the first link is copied.
- For timeouts and retries, the first link is being copied, but no callbacks are copied because the callbacks are called when the workflow times out or fails.

cc: @prasek

Reply:
Correction: timeouts and retries are handling correctly the callbacks by deferring them to the new workflow. However, in this case, there's a bug that attached callbacks are not being copied over to the new workflow, only the one in the `WorkflowExecutionStarted` event is being copied. I'll fix this asap.

Thanks @pdoerner!

Reply:
fixed with https://github.com/temporalio/documentation/pull/3472/commits/81a022ab2219cf2805fc7da8bbec72c07651da56